addon/adapters/twitch-channel-feed.js

```diff
@@ -0,0 +1,8 @@
+import TwitchAdapter from 'ember-data-twitch/adapters/twitch';
+
+export default TwitchAdapter.extend({
+
+  pathForType(/* modelName */) {
+    return 'feed';
```

Comment:
I love that idea. I [updated the root logic](https://github.com/elwayman02/ember-data-twitch/blob/79b8c45a7ec8bf7d0300c7cd9025cbf5b43aa352/addon/adapters/twitch.js#L24) and was able to just set `pluralizePath` to false on the `twitch-chat` adapter. For `channel-feed` -- which needs to be singularized and just made 'feed' -- I still left in the full override. Overall, though, the logic feels much cleaner.